libs/media-player/src/lib/as-two-digit-string.ts

```diff
@@ -0,0 +1,9 @@
+import { isInteger } from '@coscrad/validation-constraints';
+
+export const asTwoDigitString = (inputNumber: number) => {
+    const inputAsString = inputNumber.toString();
+
+    if (!isInteger(inputNumber)) return inputAsString;
```

Comment:
Good!

Reply:
But we might want to be careful with the  name of the function, though. It's really `asTwoDigitStringIfIntegerOtherwiseAsString`, which doesn't really roll off the tongue.